### ISSUE TITLE
Further improve python API client docs for get_by_id

### ISSUE THREAD

**Issue by uermel** [chanzuckerberg/cryoet-data-portal#1381]
The python API client now has `Model`-specific docs for every `Model` it supports. While the examples for `get_by_id` do contain working code, it would be great if the queries they imply would actually return data instead of `None`. This is done in the case of `find` to some extent.

Right now, for the majority of `Models`, the example in `Model.get_by_id` uses `id=1`, which will return None for most models, because that ID in most cases is not used. 

To reproduce: 
```python
import sys
import inspect
import cryoet_data_portal as cdp

classes = [obj for name, obj in inspect.getmembers(sys.modules[cdp._models.__name__], inspect.isclass)
          if obj.__module__ is cdp._models.__name__]

client = cdp.Client()

for c in classes:
    ret = c.get_by_id(client, 1) is None
    print(f"{c.__name__}.get_by_id(client, 1) is None == {ret}")
```

Returns: 
```
Alignment.get_by_id(client, 1) is None == True
Annotation.get_by_id(client, 1) is None == True
AnnotationAuthor.get_by_id(client, 1) is None == True
AnnotationFile.get_by_id(client, 1) is None == True
AnnotationMethodLink.get_by_id(client, 1) is None == True
AnnotationShape.get_by_id(client, 1) is None == True
Dataset.get_by_id(client, 1) is None == True
DatasetAuthor.get_by_id(client, 1) is None == True
DatasetFunding.get_by_id(client, 1) is None == True
Deposition.get_by_id(client, 1) is None == True
DepositionAuthor.get_by_id(client, 1) is None == False
DepositionType.get_by_id(client, 1) is None == True
Frame.get_by_id(client, 1) is None == True
FrameAcquisitionFile.get_by_id(client, 1) is None == True
GainFile.get_by_id(client, 1) is None == True
PerSectionAlignmentParameters.get_by_id(client, 1) is None == False
Run.get_by_id(client, 1) is None == True
TiltSeries.get_by_id(client, 1) is None == True
Tomogram.get_by_id(client, 1) is None == True
TomogramAuthor.get_by_id(client, 1) is None == True
TomogramVoxelSpacing.get_by_id(client, 1) is None == True
```

**Comment by andy-sweet** [comment]
If I remember correctly, some of the IDs may not be stable (i.e. they may change over time), so manually defining these once is fragile.

It might be possible for the code generator to find the first instance of each model type and use that ID value, but that would also need to be run any time those IDs actually change, so might not be much better (unless we only expect ID value changes to coincide with API releases, in which case we could just re-run code generation at release time).

**Comment by uermel** [comment]
That's true, but at least for Depositions, Datasets, Runs, and Tomograms I believe IDs should be pretty stable by now. I think in those cases (which I expect would be the most frequently used) we could use working IDs.

**Comment by melissawm** [comment]
@dgmccart had mentioned you might have a number that works everytime (something like 10000) but I can't reproduce that. I also looked at the test_get_by_id.py file which lists 20001 for the Dataset object but that doesn't seem to work for me either?? 

Any tips on how to pick those ids?

**Comment by dgmccart** [comment]
Let's use ID = 10000

This captures most of the important classes:
`Alignment.get_by_id(client, id) is None == False
Annotation.get_by_id(client, id) is None == False
AnnotationAuthor.get_by_id(client, id) is None == False
AnnotationFile.get_by_id(client, id) is None == False
AnnotationMethodLink.get_by_id(client, id) is None == True
AnnotationShape.get_by_id(client, id) is None == False
Dataset.get_by_id(client, id) is None == False
DatasetAuthor.get_by_id(client, id) is None == True
DatasetFunding.get_by_id(client, id) is None == True
Deposition.get_by_id(client, id) is None == False
DepositionAuthor.get_by_id(client, id) is None == True
DepositionType.get_by_id(client, id) is None == False
Frame.get_by_id(client, id) is None == True
FrameAcquisitionFile.get_by_id(client, id) is None == False
GainFile.get_by_id(client, id) is None == True
PerSectionAlignmentParameters.get_by_id(client, id) is None == False
PerSectionParameters.get_by_id(client, id) is None == False
Run.get_by_id(client, id) is None == False
TiltSeries.get_by_id(client, id) is None == False
Tomogram.get_by_id(client, id) is None == False
TomogramAuthor.get_by_id(client, id) is None == False
TomogramVoxelSpacing.get_by_id(client, id) is None == False`
